package-lock.json

```diff
@@ -0,0 +1,3436 @@
+{
```

Comment:
`package-lock.json` is supposed to be committed to git as it asserts what version of packages one should use to build, including transitive dependencies.  Having it in git allows for more reproducible builds.  The downside is that it means when you update a library you end up touching this file (which is generated) pretty heavily.

Normally I'm against committing generated files to git, but in this case the file is generated just so it can be shared among developers/CI so everyone builds with the same versions.